src/components/proposal/ProposalTableUser.tsx

```diff
@@ -86,7 +86,6 @@ const ProposalTableUser: React.FC = () => {
         searchQuery={sendUserProposalRequest}
         isLoading={loading}
       />
-      <br />* Pre-submission reference. Reference will change upon submission.
```

Comment:
Should it be completely removed for users table or what was the point adding it there maybe STFC needed it.